README.md

```diff
@@ -46,3 +46,12 @@ May also function as reference Service Provider implementation.
 
     openssl req -newkey rsa:2048 -nodes -keyout config/server.key \
       -x509 -out config/server.crt -config config/openssl.conf
+
+
+## Public domain
+
+This project is in the worldwide [public domain](LICENSE.md). As stated in [CONTRIBUTING](CONTRIBUTING.md):
```

Comment:
I don't really understand how this information is supposed to be split between README/CONTRIBUTING and LICENSE.

For sure LICENSE is the authoritative source, right?   So its good that README references LICENSE.   But why does it say "As stated in CONTIBUTING"?  Why not say "As stated in LICENSE"?  Or "as stated right here below" :)